internal/querycoord/impl.go

```diff
@@ -145,20 +160,48 @@ func (qc *QueryCoord) ShowCollections(ctx context.Context, req *querypb.ShowColl
 func (qc *QueryCoord) LoadCollection(ctx context.Context, req *querypb.LoadCollectionRequest) (*commonpb.Status, error) {
 	collectionID := req.CollectionID
 	//schema := req.Schema
-	log.Debug("loadCollectionRequest received", zap.String("role", typeutil.QueryCoordRole),
-		zap.Int64("msgID", req.Base.MsgID), zap.Int64("collectionID", collectionID),
-		zap.Stringer("schema", req.Schema))
+	log.Debug("loadCollectionRequest received",
+		zap.String("role", typeutil.QueryCoordRole),
+		zap.Int64("collectionID", collectionID),
+		zap.Int64("msgID", req.Base.MsgID))
+
 	status := &commonpb.Status{
 		ErrorCode: commonpb.ErrorCode_Success,
 	}
 	if qc.stateCode.Load() != internalpb.StateCode_Healthy {
 		status.ErrorCode = commonpb.ErrorCode_UnexpectedError
-		err := errors.New("query coordinator is not healthy")
+		err := errors.New("QueryCoord is not healthy")
 		status.Reason = err.Error()
-		log.Debug("load collection end with query coordinator not healthy")
+		log.Warn("load collection failed", zap.String("role", typeutil.QueryCoordRole), zap.Int64("msgID", req.Base.MsgID), zap.Error(err))
 		return status, nil
```

Comment:
using err as a return value

Reply:
error is only used to receive grpc information